charts/pega/values-large.yaml

```diff
@@ -16,7 +16,7 @@ global:
     execute: "deploy"
   # Add custom certificates to be mounted to container
   # to support custom certificates as plain text, pass certificates to certificates parameter
-  # to support multiple custom certificates as external secrets, specify each of your external secrets as an array of comma-separated strings using the certificatesSecrets parameter.
```

Comment:
the change is fine if you really have to save space, but I don't think it's clear using certs = certificates...whatever you decide, please make the terms uniform for all of the options...and passing certificates directly isn't a good idea, is it? Don't we say somewhere else to encrypt the certificate if you aren't using secrets? I'd remind them here to encrypt.

Add custom certs to be mounted to the container
to support custom certs as plain text (less secure), pass them directly to certificates parameter;
to support multiple custom certs as external secrets, specify each of your external secrets as an array of comma-separated strings using the certificatesSecrets parameter.

Reply:
Reduced certificates to certs as the line length was exceeding 180 characters , which is not allowed.

Can you please suggest some alternative for below sentence , with reduced characters.. ? will update accordingly
"to support multiple custom certificates as external secrets, specify each of your external secrets as an array of comma-separated strings using the certificatesSecrets parameter."

Reply:
Fixed with below changes

# Add custom certificates to be mounted to the container
# to support custom certificates as plain text (less secure), pass them directly using the certificates parameter;
# to support multiple custom certificates as external secrets, specify each of your external secrets
# as an array of comma-separated strings using the certificatesSecrets parameter.